src/GridFS/StreamWrapper.php

```diff
@@ -127,7 +124,6 @@ public function stream_eof(): bool
     public function stream_open(string $path, string $mode, int $options, ?string &$openedPath): bool
     {
         $this->initProtocol($path);
-        $this->mode = $mode;
```

Comment:
Introduced in https://github.com/mongodb/mongo-php-library/commit/e2bf1d45085ab22df90c03d922f1cc99d28d3077#diff-87d70a08d4e98fae61febf5131732ba9564f41ce431a1ea8f4bd759df70c2365R95 and never removed in https://github.com/mongodb/mongo-php-library/commit/bc98d9ccbe30dd7bb4cc4b0ca5104ef3b1d621a2 when `stream_stat()` was refactored.